client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx

```diff
@@ -354,8 +359,12 @@ class TransferDomainPrecheck extends React.PureComponent {
 const recordNextStep = ( domain_name, show_step ) =>
 	recordTracksEvent( 'calypso_transfer_domain_precheck_step_change', { domain_name, show_step } );
 
-const recordContinueButtonClick = domain_name =>
-	recordTracksEvent( 'calypso_transfer_domain_precheck_continue_click', { domain_name } );
+const recordContinueButtonClick = ( domain_name, losing_registrar, losing_registrar_iana_id ) =>
```

Comment:
Oooh, I just saw we have these `snake_case` vars. We shouldn't. :)

Please use `camelCase` in Calypso always. Both events. Sorry. Unless we use it weirdly elsewhere, let's try to be consistent.

Reply:
Edit: Disregard me... I see this everywhere. I don't know anymore what's right...

Reply:
Well - accodring to the tracks naming convention properties have to be in `snake_case` (see https://fieldguide.automattic.com/tracks/tracks-for-developers-product-teams/tracks-naming-conventions/ ). However I agree we should not have `snake_case` variables so we'd better construct the event properties as `{ domain_name: domainName, losing_registrar: losingRegistrar }`

Reply:
Let's do this in a separate PR? I'll go through all the tracks events for incoming transfers and fix it